.prettierignore

```diff
@@ -1,11 +1,28 @@
+# config
+.DS_Store
+.gitignore
+.prettierignore
```

Comment:
@chrisdrackett Can we replace many of these lines with a single `.*` line to ignore all dot-files & dot-folders? That would also cover IDE-related folders like the `.idea` folder Jetbrains WebStorm creates. Currently prettier is picking up some files in that folder and complains that the file type is unsupported.